src/components/Query.ts

```diff
@@ -41,18 +44,27 @@ export class Engine {
     newNodeFn: (
       name: string,
       outputChannel: OutputChannel,
-      iConn: InfluxDBConnection
-    ) => INode
+      iConn: InfluxDBConnection,
+      parent?: string
+    ) => INode,
+    v2mode?: boolean,
```

Comment:
Could we provide default values for v2mode and pp rather than making them optional?